botpress/botpress
# feat(channel-web): add support for voice messages

This PR adds voice messages support for the channel-web (using google-speech as backend for audio recognition/production).

To test this functionality:

1. Enable the google-speech module (make sure to follow its documentation)
2. Enable the voice composer for the channel-web (`enableVoiceComposer: true`)
3. Open the webchat for your bot and make sure to give the Web page access to your microphone

#### TODO:

 - [x] Autoplay voice messages only on new bot messages 
 - [x] Test with all supported browsers (firefox, opera, safari, edge)
 - [x] Autoplay messages one by one in a message group
